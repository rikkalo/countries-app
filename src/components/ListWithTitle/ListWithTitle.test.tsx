import { shallow } from "enzyme";
import React from "react";

import { ListWithTitle } from "./index";

describe("ListWithTitle", () => {
  it("should render correctly", () => {
    const wrapper = shallow(
      <ListWithTitle title="Example list">
        <div>1</div>
        <div>2</div>
        <div>3</div>
      </ListWithTitle>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
