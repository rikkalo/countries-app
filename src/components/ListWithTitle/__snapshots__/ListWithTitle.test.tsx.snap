// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListWithTitle should render correctly 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Typography))
    component="h5"
    variant="h5"
  >
    Example list
    :
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(List))>
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
  </WithStyles(ForwardRef(List))>
</Fragment>
`;
